org.opent2t.sample.hub.superpopular/org.opent2t.sample.hub.superpopular.raml

```diff
@@ -99,3 +99,37 @@ traits:
                 }
              }
     
+/subscribe:
+  description: |
+    This resource manages subscriptions to platoforms connected to this hub.
+  displayName: Subscribe
+    
+  post:
+    description: |
+       Verifies a subscription to a platform on this hub.
+    body:
+      application/json:
+        schema: Subscription
+        example: |
+          {
+            "callbackUrl": "http://myserver.com",
```

Comment:
How is the callbackurl here in the POST different from the response ? why does the server send back the same url in the response body to the POST?